source/includes/APIReference/Users/_create.md.erb

```diff
@@ -19,6 +19,10 @@ _Pass an array of user objects as the value to a 'data' property (see example)._
 
 For a full list of properties that may be set on a user, see [the User object](#the-user-object).
 
+<aside class="notice">
+By default, users will be created with the time_tracking permission set to false. Requests to create a timesheet for users in this state will result in a <code class="standout">403: Forbidden. TS-1006: User is not allowed to track time</code> HTTP response.
+</aside>
```

Comment:
This looks great.  I think it would help to also include something like:

To enable a user for time_tracking, ensure you're setting the time_tracking permission for the user at create time.